from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(678, 515)
        self.centralwidget = QtWidgets.QWidget(parent=MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(parent=self.centralwidget)
        self.label.setGeometry(QtCore.QRect(210, 50, 301, 41))
        font = QtGui.QFont()
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setStyleSheet("\n"
"background-color: rgb(255, 193, 227);\n"
"")
        self.label.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(70, 110, 91, 31))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.lineEditChisocu = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.lineEditChisocu.setGeometry(QtCore.QRect(210, 110, 301, 31))
        font = QtGui.QFont()
        font.setPointSize(15)
        self.lineEditChisocu.setFont(font)
        self.lineEditChisocu.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.lineEditChisocu.setObjectName("lineEditChisocu")
        self.lineEditChisomoi = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.lineEditChisomoi.setGeometry(QtCore.QRect(210, 150, 301, 31))
        font = QtGui.QFont()
        font.setPointSize(15)
        self.lineEditChisomoi.setFont(font)
        self.lineEditChisomoi.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.lineEditChisomoi.setObjectName("lineEditChisomoi")
        self.label_3 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(70, 150, 101, 31))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.lineEditSoho = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.lineEditSoho.setGeometry(QtCore.QRect(210, 190, 301, 31))
        font = QtGui.QFont()
        font.setPointSize(15)
        self.lineEditSoho.setFont(font)
        self.lineEditSoho.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.lineEditSoho.setObjectName("lineEditSoho")
        self.label_4 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(70, 190, 131, 31))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(70, 230, 131, 31))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_5.setFont(font)
        self.label_5.setObjectName("label_5")
        self.lineEditSokWhdung = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.lineEditSokWhdung.setGeometry(QtCore.QRect(210, 230, 301, 31))
        font = QtGui.QFont()
        font.setPointSize(15)
        self.lineEditSokWhdung.setFont(font)
        self.lineEditSokWhdung.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.lineEditSokWhdung.setObjectName("lineEditSokWhdung")
        self.label_6 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_6.setGeometry(QtCore.QRect(70, 270, 131, 31))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_6.setFont(font)
        self.label_6.setObjectName("label_6")
        self.labelTienthu = QtWidgets.QLabel(parent=self.centralwidget)
        self.labelTienthu.setGeometry(QtCore.QRect(210, 270, 301, 41))
        font = QtGui.QFont()
        font.setPointSize(16)
        self.labelTienthu.setFont(font)
        self.labelTienthu.setStyleSheet("background-color: rgb(137, 247, 255);")
        self.labelTienthu.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.labelTienthu.setObjectName("labelTienthu")
        self.pushButtonSHBT = QtWidgets.QPushButton(parent=self.centralwidget)
        self.pushButtonSHBT.setGeometry(QtCore.QRect(210, 320, 101, 41))
        self.pushButtonSHBT.setObjectName("pushButtonSHBT")
        self.pushButtonKinhDoanh = QtWidgets.QPushButton(parent=self.centralwidget)
        self.pushButtonKinhDoanh.setGeometry(QtCore.QRect(310, 320, 101, 41))
        self.pushButtonKinhDoanh.setObjectName("pushButtonKinhDoanh")
        self.pushButtonSanXuat = QtWidgets.QPushButton(parent=self.centralwidget)
        self.pushButtonSanXuat.setGeometry(QtCore.QRect(410, 320, 101, 41))
        self.pushButtonSanXuat.setObjectName("pushButtonSanXuat")
        self.pushButtonThoat = QtWidgets.QPushButton(parent=self.centralwidget)
        self.pushButtonThoat.setGeometry(QtCore.QRect(410, 360, 101, 41))
        self.pushButtonThoat.setObjectName("pushButtonThoat")
        self.pushButtonTinhTiep = QtWidgets.QPushButton(parent=self.centralwidget)
        self.pushButtonTinhTiep.setGeometry(QtCore.QRect(210, 360, 101, 41))
        self.pushButtonTinhTiep.setObjectName("pushButtonTinhTiep")
        self.pushButtonHuong = QtWidgets.QPushButton(parent=self.centralwidget)
        self.pushButtonHuong.setGeometry(QtCore.QRect(310, 360, 101, 41))
        self.pushButtonHuong.setObjectName("pushButtonHuong")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(parent=MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 678, 36))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(parent=MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        self.pushButtonTinhTiep.clicked.connect(self.lineEditChisocu.clear) # type: ignore
        self.pushButtonTinhTiep.clicked.connect(self.lineEditChisomoi.clear) # type: ignore
        self.pushButtonTinhTiep.clicked.connect(self.labelTienthu.clear) # type: ignore
        self.pushButtonTinhTiep.clicked.connect(self.lineEditSoho.clear) # type: ignore
        self.pushButtonTinhTiep.clicked.connect(self.lineEditSokWhdung.clear) # type: ignore
        self.pushButtonThoat.clicked.connect(MainWindow.close) # type: ignore
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Hoài Phương "))
        self.label.setText(_translate("MainWindow", "<html><head/><body><p><span style=\" color:#0012ff;\">PHẦN MỀM QUẢN LÝ CHI TIÊU</span></p></body></html>"))
        self.label_2.setText(_translate("MainWindow", "Chỉ số cũ:"))
        self.lineEditChisocu.setText(_translate("MainWindow", "12"))
        self.lineEditChisomoi.setText(_translate("MainWindow", "345"))
        self.label_3.setText(_translate("MainWindow", "Chỉ số mới:"))
        self.lineEditSoho.setText(_translate("MainWindow", "1.4"))
        self.label_4.setText(_translate("MainWindow", "Số hộ(SHBT):"))
        self.label_5.setText(_translate("MainWindow", "Số kWh dùng:"))
        self.lineEditSokWhdung.setText(_translate("MainWindow", "333"))
        self.label_6.setText(_translate("MainWindow", "Số tiền sẽ thu"))
        self.labelTienthu.setText(_translate("MainWindow", "Tiền SHBT =580046(VNĐ)"))
        self.pushButtonSHBT.setText(_translate("MainWindow", "SHBT"))
        self.pushButtonKinhDoanh.setText(_translate("MainWindow", "Kinh doanh"))
        self.pushButtonSanXuat.setText(_translate("MainWindow", "Sản xuất"))
        self.pushButtonThoat.setText(_translate("MainWindow", "Thoát"))
        self.pushButtonTinhTiep.setText(_translate("MainWindow", "Tính tiếp"))
        self.pushButtonHuong.setText(_translate("MainWindow", "Hướng dẫn"))
